src/lib.rs

```diff
@@ -31,6 +33,8 @@ impl LightningNode {
         config: Config,
         redundant_storage_callback: Box<dyn RedundantStorageCallback>,
     ) -> Result<Self, InitializationError> {
+        let _rt = AsyncRuntime::new()?;
```

Comment:
We could already store the runtime in `LightningNode`. Otherwise, the runtime is dropped when `new()` returns.